library.json

```diff
@@ -19,6 +19,7 @@
   "platforms": [
     "atmelavr",
     "espressif8266",
+    "espressif32",
```

Comment:
Could you append 'esp32' also to architectures in library.properties file?

Reply:
I've updated the library.properties files as well.